troposphere/ecs.py

```diff
@@ -26,6 +26,20 @@ class DeploymentConfiguration(AWSProperty):
     }
 
 
+class PlacementConstraint(AWSProperty):
+    props = {
+        'Type': (basestring, True),
+        'Expression': (basestring, False),
+    }
+
+
+class PlacementStrategy(AWSProperty):
+    props = {
+        'Type': (basestring, True),
```

Comment:
Looks like we both added this, so I closed mine (#707). You could add validation on the `Type` field here that it's one of the allowed `random`, `spread`, `binpack`.